firestore/src/android/firestore_android.cc

```diff
@@ -260,14 +260,16 @@ Local<LoadBundleTaskInternal> CreateLoadBundleTask(Env& env,
 
 const char kApiIdentifier[] = "Firestore";
 
-FirestoreInternal::FirestoreInternal(App* app) {
+FirestoreInternal::FirestoreInternal(App* app, const char* database_id) {
   FIREBASE_ASSERT(app != nullptr);
   if (!Initialize(app)) return;
   app_ = app;
 
   Env env = GetEnv();
   Local<Object> platform_app(env.get(), app_->GetPlatformApp());
-  Local<Object> java_firestore = env.Call(kGetInstance, platform_app);
+  Local<String> java_path = env.NewStringUtf(database_id);
```

Comment:
nit: could you rename `java_path` to `java_database_id`.

Reply:
Last one, then I will approve.

Reply:
This is somehow missed while committing